tests/sklearn/test_datasets.py

```diff
@@ -275,3 +275,11 @@ def test_onehot_transformer():
     dum = pd.get_dummies(X)
     assert ohe.fit_transform(X).shape[1] == dum.shape[1] == 63
     assert dum.columns.symmetric_difference(ohe.get_feature_names_out()).empty
+
+def test_openML_store():
+    """Test to verify if the new class OpenMLStore returns the same df as before"""
+    store = OpenMLStore()
+    df_new = store.download(None, None)
+    df_old = fetch_openml(data_id=1558 if percent10 else 1461, data_home=data_home
```

Comment:
Where is `percent10` and `data_home` defined in test_datasets.py? I think we can just initialize these to default values for testing.

Reply:
No, they were defined in the actual file. I will change them to default values in the testing